mw-webapp/src/component/editableTextarea/editableTextarea.tsx

```diff
@@ -53,7 +57,6 @@ export const EditableTextarea = (props: EditableTextareaProps) => {
   useEffect(() => {
     setText(props.text);
   }, [props.text]);
-
```

Comment:
Why?

Reply:
done

Reply:
So did you decided delete this line? Then why? If not - please add empty line )))